packages/discord.js/src/managers/ApplicationCommandPermissionsManager.js

```diff
@@ -237,59 +243,66 @@ class ApplicationCommandPermissionsManager extends BaseManager {
    * @returns {Promise<ApplicationCommandPermissions[]>}
    * @example
    * // Remove a user permission from this command
-   * guild.commands.permissions.remove({ command: '123456789012345678', users: '876543210123456789' })
+   * guild.commands.permissions.remove({
+   *  command: '123456789012345678', users: '876543210123456789', token: 'TotallyRealToken',
+   * })
    *   .then(console.log)
    *   .catch(console.error);
    * @example
    * // Remove multiple roles from this command
    * guild.commands.permissions.remove({
-   *   command: '123456789012345678', roles: ['876543210123456789', '765432101234567890']
+   *   command: '123456789012345678', roles: ['876543210123456789', '765432101234567890'], token: 'TotallyRealToken',
    * })
    *    .then(console.log)
    *    .catch(console.error);
    */
-  async remove({ guild, command, users, roles }) {
-    const { guildId, commandId } = this._validateOptions(guild, command);
-    if (!commandId) throw new TypeError('INVALID_TYPE', 'command', 'ApplicationCommandResolvable');
+  async remove({ guild, command, users, roles, channels, token } = {}) {
```

Comment:
Honestly not sure why we're using resolvables here, they do need to unfortunately remain split due to the singular edge case of channel id being the same as guild id, which is the same as the everyone role id